### DISCUSSION TITLE
using git_fs/git_pillar with saltclass

### DISCUSSION THREAD

**Discussion by flipfloptech** [saltstack/salt#65755] [Q&A] [answered]
Does anyone know if and if so how would one go about configuring salt to use git_fs and git_pillar to build the over all saltstack filesystem, but then run saltclass on top of it? We have a pretty big dependency on saltclass/reclass but want to also leverage git_fs git_pillar to help with management of the states and pillars themselves.

**Comment by whytewolf** [comment, marked as answer]
what you are asking is not currently possible. saltclass does not talk to the salt filesystem at all and pillar doesn't really have a filesystem. 

git_pillar is just taking the pillar roots, and telling it use this github repo to render files.

**Comment by flipfloptech** [comment]
I thought as much, I appreciate the response as confirmation!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
